### PR TITLE
VolumeWad: Return version number for WADs

### PR DESCRIPTION
**Pull request by Stevoisiak**
Thanks to JosJuice for helping me with this.

### CODE REVIEW

**Comment by Buddybenj** [comment]
LGTM

File: Source/Core/DiscIO/VolumeWad.cpp [hunk 1]
```diff
@@ -103,6 +103,17 @@ bool CVolumeWAD::GetTitleID(u8* _pBuffer) const
 	return true;
 }
 
+int CVolumeWAD::GetRevision() const
+{
+	u32 Offset = ALIGN_40(m_hdr_size) + ALIGN_40(m_cert_size) + ALIGN_40(m_tick_size);
```
Comment (lioncash) [inline Source/Core/DiscIO/VolumeWad.cpp]: lowercase `offset`

Comment (Stevoisiak) [inline Source/Core/DiscIO/VolumeWad.cpp]: All the other VolumeWad code uses a capitalized Offset variable

Comment (lioncash) [inline Source/Core/DiscIO/VolumeWad.cpp]: Yes, much of DiscIO was prior to the introduction of the coding style.

Comment (Stevoisiak) [inline Source/Core/DiscIO/VolumeWad.cpp]: I'll handle the variable capitalization in a separate PR.

**Comment by lioncash** [comment]
You can just squash these two commits into one, since it doesn't change unrelated existing code.

**Comment by JosJuice** [comment]
LGTM

**Pull request merged**